### ISSUE TITLE
lng dev not working

### ISSUE THREAD

**Issue by MattiaPontonioKineton** [WebPlatformForEmbedded/Lightning-CLI#4]
Actually, running `lng dev`:

- The first time it works, the app builds and the view is rendered correctly:
```
√ Removing "dist" folder
 
√ Ensuring "dist" folder exists
 
√ Copying support files to "dist"

√ Copying static assets to "dist"

√ Copying settings.json "dist"

√ Copying metadata.json "dist"

√ Building ES6 appBundle and saving to "dist"

√ Building ES5 appBundle and saving to "dist"
Starting up http-server, serving ./dist
Available on:
  http://192.168.0.103:8080
  http://127.0.0.1:8080
Hit CTRL-C to stop the server
open: http://127.0.0.1:8080
```
- When you save, console shows:
```
√ Ensuring "dist" folder exists

Reload your webbrowser to see the changes
```
but the changes is not correctly showed after refreshing browser.
You have to run `lng build; lng serve;` to correctly rebuild and re render the app.

### Proposal
combine `lng build; lng serve;` in a single npm script command out of the box.

### Side issue
`lng dev` is mentioned in 2 section in the readme, should be the Doc section reference to anorther command or is it right?

**Comment by michielvandergeest** [comment]
@MattiaPontonioKineton thanks for using Lightning-CLI and filing this issue!

First off, the CLI is still a bit WIP (and not _officially_ released yet). So there might be some issues with it still. We're working towards a release asap.

With regard to your issue, I'm not able to replicate this with the latest version (#dbd2c3a). Could you try and update to the latest version and see if perhaps it has been fixed somewhere in the last days?

I've corrected your side issue about `lng dev` being mentioned twice in the readme by the way. Thanks :)

**Comment by MattiaPontonioKineton** [comment]
Hello @michielvandergeest,

steps to reproduce:

0. `npm install -g WebPlatformForEmbedded/Lightning-CLI`
1. `lng -V` => `Lightning-CLI 1.2.1`
2. `lng create`
3. Change `src/App.js`:
```
import { Lightning } from 'wpe-lightning-sdk'

export default class App extends Lightning.Component {
  static _template() {
    return {
      w: 1920,
      h: 1080,
      text: {
        text: 'HELLO WORLD!',
        textColor: 0xff00ffff
      }
    }
  }
}
```
4. `lng dev`
```
√ Removing "dist" folder

√ Ensuring "dist" folder exists
 
- Copying support files to "dist"

⚠️  You are using an older version of the Lightning SDk. Please consider upgrading to the latest version.  ⚠️

√ Copying support files to "dist"

√ Copying static assets to "dist"

√ Copying settings.json "dist"

√ Copying metadata.json "dist"
 
√ Building ES6 appBundle and saving to "dist"

√ Building ES5 appBundle and saving to "dist"
Starting up http-server, serving ./dist
Available on:
  http://10.41.18.127:8086
  http://192.168.0.107:8086
  http://127.0.0.1:8086
Hit CTRL-C to stop the server
open: http://127.0.0.1:8086
```
5. Change `src/App.js`:
```
import { Lightning } from 'wpe-lightning-sdk'

export default class App extends Lightning.Component {
  static _template() {
    return {
      w: 1920,
      h: 1080,
      text: {
        text: 'HELLO WORLD! CHANGED',
        textColor: 0xff00ffff
      }
    }
  }
}
```
```
√ Ensuring "dist" folder exists

Reload your webbrowser to see the changes
```
6. Reload the browser
---
The Text Texture remains `HELLO WORLD!` as the first time. The app do not rebuild.

---
Is Lightning-CLI 1.2.1 the latest version (#dbd2c3a)?

Best regards

**Comment by michielvandergeest** [comment]
After further investigation this turned out to be a windows specific bug, that has been fixed in v 1.3.0
